src/diagnostics/default_diagnostics.jl

```diff
@@ -107,71 +107,71 @@ function default_diagnostics(
 ) where {FT}
 
     define_diagnostics!(land_model)
-
+    possible_diags = [
```

Comment:
I had implemented this change for the LandModel in another PR - it speeds up our fluxnet runs by ~30% to use save the diagnostics we need. In this PR, I made the same change for the soil canopy model in order to run the ozark_pft simulation with the same code as the run_fluxnet simulation.